src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -782,4 +792,20 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         }
         return result;
     }
+    
+    public List<String> getArchivedCourseIds(List<CourseDetailsBundle> allCourses, List<InstructorAttributes> instructorList) {
+        List<String> activeCourseIds = new ArrayList<String>();
+        for (CourseDetailsBundle cdb : allCourses) {
+            if (cdb.course.isArchived) {
```

Comment:
Thanks.
@LowWeiLin since we will remove `isArchived` from the course eventually, I think we should remove the check for `course.isArchived` from `CoursesLogic` as well.